test/typings/tsconfig.5.2.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./tsconfig.4.7.json",
+  "extends": "./tsconfig.json",
```

Comment:
Let's tidy this up a bit. The implementation is that if there's no designated config for a TS version it grabs the default one. I did it so there be no repetitive configs.